index.html

```diff
@@ -8,8 +8,20 @@
     <link rel="stylesheet" href="css/index.css">
 </head>
 <body>
+    <!-- Header -->
+    <header>
     <h1>Haregebezu Amare</h1>
```

Comment:
Formatting: indent this line now that it's wrapped.

Reply:
With a fixed header, you may also want an opaque background: Otherwise it can be hard to read overlapping text.